ubi8-openjdk-11-runtime.yaml

```diff
@@ -0,0 +1,50 @@
+# This is an Image descriptor for Cekit
+
+schema_version: 1
+
+from: "registry.access.redhat.com/ubi8/ubi-minimal"
+name: &name "ubi8/openjdk-11"
+version: &version "1.3"
+description: "Source To Image (S2I) image for Red Hat OpenShift providing OpenJDK 11"
+
+labels:
+- name: "io.k8s.description"
+  value: "Platform for building and running plain Java applications (fat-jar and flat classpath)"
+- name: "io.k8s.display-name"
+  value: "Java Applications"
+- name: "io.openshift.tags"
+  value: "builder,java"
+- name: "maintainer"
+  value: "Red Hat OpenJDK <openjdk@redhat.com>"
+- name: "com.redhat.component"
+  value: "openjdk-11-ubi8-container"
+- name: "usage"
+  value: "https://access.redhat.com/documentation/en-us/red_hat_jboss_middleware_for_openshift/3/html/red_hat_java_s2i_for_openshift/"
+- name: "com.redhat.license_terms"
+  value: "https://www.redhat.com/en/about/red-hat-end-user-license-agreements#UBI"
+
+envs:
+- name: PATH
+  value: $PATH:"/usr/local/s2i"
+- name: "JBOSS_IMAGE_NAME"
+  value: *name
+- name: "JBOSS_IMAGE_VERSION"
+  value: *version
+
+ports:
+- value: 8080
+- value: 8443
+
+modules:
+  repositories:
+  - path: modules
+  install:
+  - name: jboss.container.openjdk.jre
+    version: "11"
+  - name: jboss.container.java.singleton-jdk
```

Comment:
we should check whether this is necessary or not. it is in the JDK image, otherwise JDK8 and 11 are installed. If you remove it, does the image end up with just the one jRE in it?